config/manifests/bases/openstack-operator.clusterserviceversion.yaml

```diff
@@ -21,7 +21,7 @@ spec:
   customresourcedefinitions:
     owned:
     - description: OpenStackClient is the Schema for the openstackclients API
-      displayName: OpenStack Client
+      displayName: Open Stack Client
```

Comment:
Does this PR need to be rebased to pick up #1006 ?

Reply:
sure, just rebased